tests/test_read.py

```diff
@@ -0,0 +1,70 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+import random
+import string
+from unittest import mock
+import pytest
+
+from azure.core.credentials import AzureSasCredential, AzureNamedKeyCredential
+from azure.identity import DefaultAzureCredential
+
+from azstoragetorch.io import BlobIO
+from azstoragetorch._client import AzStorageTorchBlobClient
+
+from azure.core.exceptions import ResourceExistsError
+from azure.storage.blob import BlobServiceClient
+
+
+
+@pytest.fixture
+def account_url():
+    account_name = os.environ.get('PYTORCH_STORAGE_ACCOUNT_NAME', None) or "pytorchstorageaccount"
+    return "https://" + account_name.strip('"') + ".blob.core.windows.net"
+
+@pytest.fixture
+def container_name():
+    return ''.join(random.choices(string.ascii_lowercase + string.digits, k=8))
```

Comment:
For this line, it may be worth introducing it as a helper function e.g. `random_resource_name()` so that we can call it in the different fixtures that need a randomized resource name and not have to individually repeat this logic for each fixture (and future fixture) that may need it.